opensearch/README.md

```diff
@@ -90,8 +90,8 @@ This guide provides a quick and straightforward way to use **OpenSearch** as a G
 | cluster.cluster.dashboards.service.type | string | `"ClusterIP"` | dashboards service type |
 | cluster.cluster.dashboards.tls.caSecret | object | `{}` | Secret that contains the ca certificate as ca.crt. If this and generate=true is set the existing CA cert from that secret is used to generate the node certs. In this case must contain ca.crt and ca.key fields |
 | cluster.cluster.dashboards.tls.enable | bool | `false` | Enable HTTPS for dashboards |
-| cluster.cluster.dashboards.tls.generate | bool | `true` | generate certificate, if false secret must be provided |
-| cluster.cluster.dashboards.tls.secret | string | `nil` | Optional, name of a TLS secret that contains ca.crt, tls.key and tls.crt data. If ca.crt is in a different secret provide it via the caSecret field |
+| cluster.cluster.dashboards.tls.generate | bool | `false` | generate certificate, if false secret must be provided |
+| cluster.cluster.dashboards.tls.secret | object | `{"name":"opensearch-http-cert"}` | Optional, name of a TLS secret that contains ca.crt, tls.key and tls.crt data. If ca.crt is in a different secret provide it via the caSecret field |
```

Comment:
I am not understanding the if-stmt here:
> If ca.crt is in a different secret provide it via the caSecret field 

why is this only relevant for `ca.crt`? Where is this _field_ that I can set?

Reply:
When requesting a certificate via cert-manager, the ca.crt field is automatically included. If I want to use my own CA, I can provide it manually by referencing a separate secret. In our case, cert-manager adds the DigiCert CA to the certificate secret.